spec/support/controller_macros.rb

```diff
@@ -17,9 +17,9 @@ def login_curator
 
   def login_user
     before(:each) do
-      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @request.env['devise.mapping'] = Devise.mappings[:user]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.